ephios/api/views/events.py

```diff
@@ -80,9 +82,20 @@ class ShiftViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = Shift.objects.all()
 
 
+class EventFilterSet(FilterSet):
```

Comment:
maybe this should live in filters.py?

Reply:
sure, as long as it does not break https://github.com/fsr-de/pretix-helper-validation/blob/main/pretix_fsr_validation/signals.py#L136

do you prefer smth else than isoformat? that was just the default that django-filter showed me, so i went with it

Reply:
Well they'd need to change the API parameter name, but we can keep a deprecated old version of the parameter if you want

Reply:
I'm also fine with removing it, but then we should highlight the breaking change in the changelog so they don't update for now as they are in the presale phase where they rely on that API